src/configure.test.js

```diff
@@ -57,7 +57,7 @@ test('Imports are merged on pages', () => {
 test('basePath support', () => {
   const config = configure({
     title: 'Catalog',
-    basePath: 'catalog',
+    basePath: '/catalog',
```

Comment:
Is this leading slash important? If it is, it should be documented, I wouldn't know why it is important.